README.md

```diff
@@ -109,10 +123,11 @@ STATICFILES_DIRS = (
 Restart the apache server to put all changes into effect. 
 
 ```
-/etc/init.d/apache2 reload
+sudo /etc/init.d/apache2 reload
 ```
 
-###Migrate the Database Schema using South
```

Comment:
Let's make a note of that then leave it in for now. Once everything is more organized we may not need it anymore.